galaxy_ng/app/api/v3/urls.py

```diff
@@ -18,6 +18,11 @@
 
 
 urlpatterns = [
+    path(
```

Comment:
I suggest to map urls under `content/` that way it doesn’t get included when the route doesn’t have the `content/<distro>` - see #410 

https://github.com/ansible/galaxy_ng/blob/e5694720482a7a65266ee83c0054550c515dca47/galaxy_ng/app/api/urls.py#L38